README.md

```diff
@@ -12,7 +12,10 @@
 
 **Note**: The `main` branch may be in an *unstable or even broken state* during development. For stable versions, see [releases][github-release].
 
-![etcd Logo](logos/etcd-horizontal-color.svg)
+<picture>
```

Comment:
```
![etcd logo light mode](https://raw.githubusercontent.com/cncf/artwork/master/projects/etcd/horizontal/color/etcd-horizontal-color.png#gh-light-mode-only)
![etcd logo dark mode](https://raw.githubusercontent.com/cncf/artwork/master/projects/etcd/horizontal/white/etcd-horizontal-white.png#gh-dark-mode-only)
```

The CNCF provides that logo for us https://github.com/cncf/artwork/blob/main/projects/etcd/horizontal/white/etcd-horizontal-white.png

Reply:
Thanks for the review.

> I think we can try this one [github.blog/changelog/2021-11-24-specify-theme-context-for-images-in-markdown](https://github.blog/changelog/2021-11-24-specify-theme-context-for-images-in-markdown/)

This one is deprecated according to [the doc](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#specifying-the-theme-an-image-is-shown-to).

<img width="790" alt="image" src="https://github.com/user-attachments/assets/90494f1e-ac84-4bfc-9806-5b8afcda9cdd">

> The CNCF provides that logo for us

I changed it to the CNCF logo. However, I used the logo link pointing to the specific [commit](https://raw.githubusercontent.com/cncf/artwork/9870640f123303a355611065195c43ac3f27aa19/projects/etcd/horizontal/white/etcd-horizontal-white.png) instead of `main`. This way, if someone removes the logo from `cncf/artwork` or moves it to another location, the link will remain valid.